site/install.md

```diff
@@ -37,39 +37,40 @@ See the [release notes][10] for more about what's new.
 
 [Java Artifacts on Maven Central][4]
 
-## Binary Installers for C, C++, Python
+### Python Wheels
 
-### C++ and Python Conda Packages (Unofficial)
+We have provided official binary wheels on PyPI for Linux, macOS, and Windows:
+
+```shell
+pip install pyarrow=={{site.data.versions['current'].pinned_number}}
+```
+
+We recommend pinning `{{site.data.versions['current'].pinned_number}}`
+in `requirements.txt` to install the latest patch release.
```

Comment:
Can we remove this? Perhaps an artifact of a bygone unstable time? I'd think we'd want to discourage version pinning (people who want/need to pin will do so anyway).

Reply:
We can remove this when we reach 1.0.0.
API isn't stable until 1.0.0.

Reply:
https://issues.apache.org/jira/browse/ARROW-5445

I'm noticing that for every issue I pick up, I end up making two or three more 🤔

Reply:
Version pinning is commonplace in the Python world -- I don't think API stability has much to do with it (we will still have some API changes or deprecations after 1.0 I would guess)

Reply:
OK. I've closed https://issues.apache.org/jira/browse/ARROW-5445 .